text = '''Viruses evolve as a result of mutation (misincorporations, insertions or deletions, and recombination) and natural selection for favorable traits such as more efficient viral replication, transmission, and evasion of host defenses. Newly selected traits may be linked in unpredictable ways and raise concern that virus spread and evolution could result in greater virulence (disease severity). The limited diversity of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) reported during 2020, ascribed to the 3′-5′ exonuclease proofreading function of nonstructural protein 14 (nsp14), led to the view that vaccines based on a single sequence of the viral spike (S) protein, which mediates host cell entry, would likely generate immune protection to all circulating variants (1). However, variants of SARS-CoV-2 with mutations in S have emerged around the world, posing potential challenges for vaccination and antibody-based therapies. The continued spread of SARS-CoV-2 creates the opportunity for accumulation of additional consequential mutations in S and throughout the viral genome.

Although SARS-CoV-2 shares high sequence homology with SARS-CoV, which caused the 2002–2004 SARS outbreak, the coronavirus family is diverse in both sequence and in host receptor preference. For example, SARS-CoV-2 and a “common cold” human coronavirus, HCoV-NL63, both recognize angiotensin-converting enzyme 2 (ACE2) as the host cell receptor, but SARS-CoV-2 and HCoV-NL63 belong to different coronavirus genera and have major sequence and structural differences in the receptor-binding domain (RBD) of S, sharing <30% sequence homology (2). This diversity in S indicates that coronaviruses have broad potential to tolerate changes in both sequence and structure without substantial loss of function. This may partially explain why coronaviruses can undergo zoonotic transmission and suggests that the full evolutionary potential of SARS-CoV-2 has yet to be revealed.

The S protein comprises two subunits: S1, which contains the RBD, and S2, which mediates virus–host cell fusion. Antibody-neutralizing epitopes are scattered throughout S but are mostly concentrated within the RBD. Despite the potential for plasticity, after nearly a year of spread (from December 2019) to >100 million people, there was limited evidence for evolution of SARS-CoV-2 S. The only notable evolutionary event was the D614G (Asp614→Gly) substitution in S1, which increases ACE2 affinity, leading to higher infectivity and transmissibility. Viral sequences deposited in public databases were mostly obtained from the upper respiratory tract during acute infection, before major immune responses have occurred. Such sequences might not capture the effect of within-host immune selection on viral diversification.'''